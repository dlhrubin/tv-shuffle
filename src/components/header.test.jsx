import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './header';

afterEach(cleanup);

describe('Header', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment(<Header />)).toMatchSnapshot();
  });
});
