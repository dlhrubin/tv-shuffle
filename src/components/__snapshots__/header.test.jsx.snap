// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<DocumentFragment>
  <header>
    <div>
      <h1
        style="margin: 0px;"
      >
        <a
          href="/"
        />
      </h1>
    </div>
  </header>
</DocumentFragment>
`;
